src/components/SearchImagesPage.vue

```diff
@@ -20,14 +20,20 @@
         </b-tabs>
         <div class="py-3 px-3">
           <search-pills store="searchImages" v-on:remove="onRemoveFilter"/>
-          <b-media v-if="similarToImage">
+          <b-media v-if="similarToImage" class="pb-3">
             <div style="width:128px;" slot="aside">
-              <b-img fluid-grow v-bind:src="similarToImage.regions[0].iiifFragment" />
+              <b-img
+                rounded
```

Comment:
we don't use any rounded elements in the edgy application ... I would change this for consistency